sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/io/sermant/flowcontrol/retry/client/HttpUrlConnectionResponseStreamInterceptor.java

```diff
@@ -92,8 +97,19 @@ protected int getStatusCode(ExecuteContext context) {
     }
 
     @Override
-    protected boolean canInvoke(ExecuteContext context) {
-        return XdsThreadLocalUtil.isConnected() && XdsThreadLocalUtil.getScenarioInfo() != null;
+    protected boolean needRetry(io.github.resilience4j.retry.Retry retry, Object result, Throwable throwable) {
+        final RetryConfig retryConfig = retry.getRetryConfig();
+        boolean isNeedRetry = retryConfig.getExceptionPredicate().test(throwable);
+        if (isNeedRetry) {
+            final long interval = retry.getRetryConfig().getIntervalBiFunction().apply(1, null);
+            try {
```

Comment:
use retryConfig.  And what is the meaning of (1, null)

Reply:
The value 1 indicates the number of retry times. The second parameter is invalid, so the value null is transferred,